pkg/pgmodel/pgx.go

```diff
@@ -323,6 +353,78 @@ func (p *pgxInserter) getMetricTableName(metric string) (string, error) {
 	return tableName, err
 }
 
+func (p *pgxInserter) getMetricTableInserter(metric string) chan insertDataRequest {
+	var inserterI interface{}
+	var ok bool
+	var inserter chan insertDataRequest
+	if inserterI, ok = p.metricTableInserters.Load(metric); !ok {
+		ch := make(chan insertDataRequest, 1000)
+		ii, loaded := p.metricTableInserters.LoadOrStore(metric, ch)
```

Comment:
What happens if the cache evicts an item? Is the value channel closed somehow? Does BigCache have facility for this?

Reply:
This is not a cache, it's a map, my expectation is that we'll ≈never have enough dead metrics that it's worthwhile to garbage collect them.